workflow/yadage/workflow-htcondor.yaml

```diff
@@ -0,0 +1,38 @@
+# Note that if you are working on the analysis development locally, i.e. outside
+# of the REANA platform, you can proceed as follows:
+#
+#   $ mkdir -p yadage-local-run/yadage-inputs
+#   $ cd yadage-local-run
+#   $ cp -a ../code ../data yadage-inputs
+#   $ yadage-run . ../workflow/yadage/workflow.yaml -p sleeptime=0 \
+#       -p inputfile=data/names.txt \
+#       -p helloworld=code/helloworld.py \
+#       -d initdir=`pwd`/yadage-inputs
+#   $ cat helloworld/greetings.txt
+#   Hello Jane Doe!
+#   Hello Joe Bloggs!
+
+stages:
+  - name: helloworld
+    dependencies: [init]
+    scheduler:
+      scheduler_type: 'singlestep-stage'
+      parameters:
+        sleeptime: {stages: init, output: sleeptime, unwrap: True}
+        inputfile: {stages: init, output: inputfile, unwrap: true}
+        helloworld: {stages: init, output: helloworld, unwrap: true}
+        outputfile: '{workdir}/greetings.txt'
+      step:
+        process:
+          process_type: 'string-interpolated-cmd'
+          cmd: 'python "{helloworld}" --sleeptime {sleeptime} --inputfile "{inputfile}" --outputfile "{outputfile}"'
+        publisher:
+          publisher_type: 'frompar-pub'
+          outputmap:
+            outputfile: outputfile
+        environment:
+          environment_type: 'docker-encapsulated'
+          image: 'python'
+          imagetag: '2.7-slim'
+          resources:
+            - compute_backend: htcondorcern
```

Comment:
@lukasheinrich,

Is it okay to define compute backend (K8s/HTC/Slurm) requirement in this way?